ivy_tests/test_ivy/test_frontends/test_tensorflow/test_math.py

```diff
@@ -2844,6 +2844,56 @@ def test_tensorflow_softmax(
     )
 
 
+# sampled_softmax_loss
+@handle_frontend_test(
+    fn_tree="tensorflow.math.sampled_softmax_loss",
+    dtype_values_axis=helpers.dtype_values_axis(
+        available_dtypes=helpers.get_dtypes("float"),
+        min_num_dims=1,
+        valid_axis=True,
+        force_int_axis=True,
+        allow_inf=False,
+    ),
+    test_with_out=st.just(False),
+)
+def test_tensorflow_softmax(
```

Comment:
Hi @rohitsalla,
the function tensorflow_softmax already exist
you reserved the issue to implement the sampled_softmax_loss function,
please implement the function and it's test case and let me know.
Thanks